benchmark/src/test/java/com/datatorrent/benchmark/memsql/MemsqlInputBenchmarkTest.java

```diff
@@ -77,8 +77,8 @@ public void testMethod() throws SQLException, IOException
     attributeMap.put(OperatorContext.PROCESSING_MODE, ProcessingMode.AT_LEAST_ONCE);
     attributeMap.put(OperatorContext.ACTIVATION_WINDOW_ID, -1L);
     attributeMap.put(DAG.APPLICATION_ID, APP_ID);
-    OperatorContextTestHelper.TestIdOperatorContext context =
-        new OperatorContextTestHelper.TestIdOperatorContext(OPERATOR_ID, attributeMap);
+    OperatorContextTestHelper.MockOperatorContext context =
```

Comment:
Why not to use this PR that already modifies a large number of files to fix the usage? As this line needs to be changed, it should be changed to use OperatorContext interface instead of MockOperatorContext.